rst2pdf/tests/check_artifacts.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import glob
+import shutil
+
+from autotest import checkmd5
+
+# First, let's check that the existing artifacts are "good"
+
+artifacts = glob.glob('reference/*.pdf')
+
+for artifact in artifacts:
+    md5_path = artifact.replace('.pdf', '.json').replace('reference/', 'md5/')
+    res, _ = checkmd5(artifact, md5_path, [], False)
+    if res != 'good':
+        print 'BAD ARTIFACT: %s ' % res, artifact
+
+# Check if any output file is good and has no matching artifact
+
+for new_artifact in glob.glob('output/*.pdf'):
+    artifact = new_artifact.replace('output/', 'reference/')
+    md5_path = artifact.replace('.pdf', '.json').replace('reference/', 'md5/')
```

Comment:
We only need to calculate `md5_path` if the artifact doesn't already exist in the reference folder. So maybe move this line into the `else` statement?